contracts/JBMultiTerminal.sol

```diff
@@ -1541,28 +1534,6 @@ contract JBMultiTerminal is JBOperatable, Ownable, ERC2771Context, IJBMultiTermi
         emit RefundHeldFees(_projectId, _amount, refundedFees, leftoverAmount, _msgSender());
     }
 
-    /// @notice Reverts an expected payout.
-    /// @param _projectId The ID of the project having paying out.
-    /// @param _token The address of the token having its transfer reverted.
-    /// @param _expectedDestination The address the payout was expected to go to.
-    /// @param _allowanceAmount The amount that the destination has been allowed to use.
-    /// @param _depositAmount The amount of the payout as debited from the project's balance.
-    function _revertTransferFrom(
-        uint256 _projectId,
-        address _token,
-        address _expectedDestination,
-        uint256 _allowanceAmount,
-        uint256 _depositAmount
-    ) internal {
-        // Cancel allowance if needed.
-        if (_allowanceAmount != 0 && _token != JBTokens.ETH) {
```

Comment:
do we need to keep this part too @xBA5ED ?

Reply:
since we are still increasing allowance, if the transaction fails and the funds are being re-added to the balance, we should decrease the allowance.